src/app/pages/details/details.hbs

```diff
@@ -0,0 +1,172 @@
+<style scoped>
+    @import url(/app/pages/details/details.css);
+    @import url(https://fonts.googleapis.com/css?family=Lato:400,900);
+</style>
+<section class="floating-menu">
+    <div class="title">Get this title</div>
+    <div class="get-this-book"></div>
+    <div class="notes">
+        <sup>*</sup>Our web view is Section 508 Level AA accessible.
```

Comment:
I think this should be `web site` not `web view`.  Also, we're not AA accessible, we're only Level A accessible.